sunpy/visualization/animator/base.py

```diff
@@ -292,15 +298,15 @@ def _add_widgets(self):
         self.sliders = []
         self.slider_buttons = []
         for i in range(self.num_sliders):
-            x = i * 2
+            y = i * 2
```

Comment:
Is changing this to y to fix one of the issues we identified when you were here?  Or is this another issue?

Reply:
It's just because it's a row index not a column index, so it was confusing to me re-reading the code that it was `x` and not `y`.